packages/script/install.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+chmod 775 -R /opt/yugabyte-2.0.5.2
+cd /opt/yugabyte-2.0.5.2/
+
+mkdir -p /var/log/yugabyte-logs/
```

Comment:
typically databases like mysql and postgres also create a non-root user to run the database as. Does fpm take care of that for us?

Reply:
fpm don't have such option so I have added a command to create a user called `yugabyte` and changed the ownership and permission of installed directory to `yuagbyte` user.

Reply:
How about `/var/log/yugabyte` instead of `/var/log/yugabyte-logs`? Since it is in `/var/log`, the `-logs` part is redundant.